common/artifacts/000_wc_artifacts.txt

```diff
@@ -801,6 +1060,62 @@ dark_crystal = {
 
 	picture = "GFX_inv_misc_orb_03"
 }
+horn_of_cenarius = {
+	quality = 5
+	flags = { misc powerful_magical_artifact }
+
+	active = {
+		is_adult = yes
+		is_druid_class_trigger = yes
+	}
```

Comment:
I would add `religion_group = druidism_group` since Void-corrupted druids can use it right now and get opinion of Druidic followers.

Reply:
done